src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -175,35 +190,281 @@ type SingleBlastJobResultProps = {
   jobId: string;
   species: Species;
   diagramWidth: number;
+  blastDatabase: DatabaseType;
 };
 
+const hitsTableColumns: DataTableColumns = [
+  {
+    width: '0px',
+    columnId: 'id',
+    title: 'ID',
+    isFilterable: false,
+    isHideable: false
+  },
+  {
+    width: '130px',
+    columnId: 'e_value',
+    title: 'E-value',
+    isSortable: true,
+    helpText: (
+      <span>
+        The number of times a match is expected to occur by chance <br></br>
+        Typically, a low e-value indicates greater similarity between sequences
+      </span>
+    )
+  },
+  {
+    width: '100px',
+    columnId: 'hit_length',
+    title: 'Length',
+    isSortable: true
+  },
+  { width: '200px', columnId: 'view_alignment', title: '', isHideable: false },
```

Comment:
Does `title: ''` suggest that title should be an optional field?

Reply:
Changed it to be optional